server/controllers/finance/reports/reportAccounts/index.js

```diff
@@ -67,54 +62,49 @@ function document(req, res, next) {
 /**
  * @function queryReportAccount
  * This feature select all transactions for a specific account
-*/ 
-function queryReportAccount(accountId, source){
+*/
+function queryReportAccount(accountId, source) {
   source = parseInt(source);
+  let sql;
 
-  if(source === 1){
-
-    let sql = `
+  if (source === 1) {
```

Comment:
There is no indication at all here as to what source is supposed to mean or refer to - it looks like it's taking values from either the posting journal, general ledger or both however this is not at all clear given the code. 

Should this be refactored here or do you think it is inappropriate for this pull request?